src/types.ts

```diff
@@ -32,10 +32,19 @@ export enum ConfirmationType {
   Simulate = 3,
 }
 
-export const MarketNumber = {
+export const MarketId = {
```

Comment:
This is a breaking change, but I don't think anyone is using the version I just published today, so I think it's fine